Jenkinsfile

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env groovy
+
+def dockerRegistry = "329802642264.dkr.ecr.eu-west-1.amazonaws.com"
+def nodeImageVersion = "0.0.5"
+def nodeImage = "${dockerRegistry}/bbc-news/node-8-lts:${nodeImageVersion}"
+def nodeName
+def stageName = ""
+def getCommitInfo = {
+  infraGitCommitAuthor = sh(returnStdout: true, script: "git --no-pager show -s --format='%an' ${GIT_COMMIT}").trim()
+  appGitCommit = sh(returnStdout: true, script: "cd ${APP_DIRECTORY}; git rev-parse HEAD")
+  appGitCommitAuthor = sh(returnStdout: true, script: "cd ${APP_DIRECTORY}; git --no-pager show -s --format='%an' ${appGitCommit}").trim()
+  appGitCommitMessage = sh(returnStdout: true, script: "cd ${APP_DIRECTORY}; git log -1 --pretty=%B").trim()
+}
+
+pipeline {
+  agent any
+  options {
+    timeout(time: 60, unit: 'MINUTES')
```

Comment:
For what the Jenkins file is doing right now, this seems long; although it's the same length's as Simorgh.